optimum/commands/export/openvino.py

```diff
@@ -157,6 +157,16 @@ def parse_args_openvino(parser: "ArgumentParser"):
             "applying scale estimation takes additional memory and time."
         ),
     )
+    optional_group.add_argument(
+        "--gptq",
+        action="store_true",
+        default=None,
+        help=(
+            "Indicates whether to apply GPTQ algorithm that optimizes compressed weights in a layer-wise fashion to "
+            "minimize the difference between activations of a compressed and original layer. Please note, that "
+            "applying GPTQ takes additional memory and time."
+        ),
+    )
```

Comment:
doesn't it make more sense to add GPTQ as a `quant_method` like AWQ instead of creating a new field for it ?